modules/apps/transfer/client/cli/tx.go

```diff
@@ -89,7 +90,21 @@ to the counterparty channel. Any timeout set to 0 is disabled.`),
 				}
 
 				if timeoutTimestamp != 0 {
-					timeoutTimestamp = consensusState.GetTimestamp() + timeoutTimestamp
+					// use local clock time as reference time if it is later than the
+					// consensus state timestamp of the counter party chain, otherwise
+					// still use consensus state timestamp as reference
+					now := time.Now().UnixNano()
```

Comment:
I think its probably safe to convert to `uint64` here:

```suggestion
					now := uint64(time.Now().UnixNano())
```

Reply:
I know it's very unlikely, but if the clock of the machine is set to a date time before Jan 1st 1970 12:00 AM, then it could be negative and the conversion would give the wrong time, right?

Reply:
Sure, I guess its better to be overly defensive, although I don't think it would affect the chain, just panic the cli. 
I wonder if the datetime on the machine was that badly out of sync if the node would have run into problems earlier, perhaps at runtime 🤔

Reply:
Yes, I guess if the clock so much out of sync there would have been other problems already before.